frontend/src/index.tsx

```diff
@@ -1,8 +1,8 @@
-import './__generated/antd.css'
 import '@highlight-run/rrweb/dist/rrweb.min.css'
 import '@fontsource/poppins'
 import '@highlight-run/ui/styles.css'
 import './index.css'
+import './__generated/antd.css'
```

Comment:
There might not be any practical implications looking at the preview, but I think it makes more sense for our custom css to go after the generated antd.css so they can override them when necessary.

Reply:
oops yeah, was trying that - will revert!